cape/cape_main.py

```diff
@@ -2178,6 +2180,19 @@ def _extract_commands(self) -> None:
                 }
             )
 
+    def _extract_buffers(self) -> None:
+        if os.path.exists(BUFFER_PATH):
+            for entry in os.scandir(BUFFER_PATH):
+                if entry.is_file():
+                    self.artifact_list.append(
+                        {
+                            "name": entry.name,
+                            "path": entry.path,
+                            "description": "Extract buffers",
```

Comment:
We can be more specific:
```suggestion
                            "description": "PEs extracted from Windows API buffers",
```